appendices/boilerplate.txt

```diff
@@ -288,6 +288,9 @@ endif::VK_NV_win32_keyed_mutex[]
                                      | Microsoft Windows      | `vulkan_win32.h`         | `<windows.h>`                  | dname:VK_USE_PLATFORM_WIN32_KHR
 | `<<VK_KHR_xcb_surface>>`           | X11 Xcb                | `vulkan_xcb.h`           | `<xcb/xcb.h>`                  | dname:VK_USE_PLATFORM_XCB_KHR
 | `<<VK_KHR_xlib_surface>>`          | X11 Xlib               | `vulkan_xlib.h`          | `<X11/Xlib.h>`                 | dname:VK_USE_PLATFORM_XLIB_KHR
+ifdef::VK_EXT_directfb_surface[]
+| `<<VK_EXT_directfb_surface>>`      | DirectFB               | `vulkan_directfb.h`      | `<directfb.h>`                 | dname:VK_USE_PLATFORM_DIRECTFB_EXT
```

Comment:
Isn't the canonical path <directfb/directfb.h>?

Reply:
The warning may be fixed with https://github.com/KhronosGroup/Vulkan-Docs/pull/1302

In the Linux filesystem hierarchy standard, headers are located in /usr/include and DirectFB headers are installed in the /usr/include/directfb directory (/usr/include/directfb/directfb.h)
When compiling DirectFB applications, it is necessary to add this include directory with -I/usr/include/directfb, so #include <directfb.h> is used in a standard DirectFB code. That's why I put <directfb.h> in appendices/boilerplate.txt : is it wrong ?

Nicolas Caramelli

Reply:
IDK, just inconsistent. E.g. Vulkan is installed to `/usr/include/vulkan/vulkan.h` and canonical way to include it is `<vulkan/vulkan.h>`.

Reply:
OK, I made the update.